Bottom: 471f2fbacd83e5f54dad4b42c7c18714d4beface
Top:    a6ec9963ffbb3683e2284c21c3635701f68048dc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-06-13 01:59:10 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
